controlplane/api/v1beta1/rke2controlplane_webhook.go

```diff
@@ -48,6 +48,16 @@ var _ webhook.Defaulter = &RKE2ControlPlane{}
 // Default implements webhook.Defaulter so a webhook will be registered for the type.
 func (r *RKE2ControlPlane) Default() {
 	bootstrapv1.DefaultRKE2ConfigSpec(&r.Spec.RKE2ConfigSpec)
+
+	// (Quirk)
```

Comment:
can we remove the comments?

Reply:
```suggestion
	// Defaults MachineTemplate.InfrastructureRef to Spec.InfrastructureRef, if missing
```
seems to be more formal